File: src/main/scala/analysis/advertising/bean/AdUserClickCount.scala
package analysis.advertising.bean

/**
 * 广告实时统计
 *
 * @param date
 * @param userid
 * @param adid
 * @param clickCount
 */
case class AdUserClickCount(date: String,
                            userid: Long,
                            adid: Long,
                            clickCount: Long)

File: src/main/java/com/jxust/entity/AdUserClickCount.java
package com.jxust.entity;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
* Create by Jayden in 2020/3/7
*/

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AdUserClickCount implements Serializable {
    private String date;

    private Integer userid;

    private Integer adid;

    private Integer clickcount;

    private static final long serialVersionUID = 1L;
}